### PR TITLE
FIX: Use absolute imports in testing,tests and segment files

### PR DESCRIPTION
**Pull request by souravsingh**
Changed some of the relative imports in testing and tests to use absolute imports.
Fixes a small part of https://github.com/nipy/dipy/issues/969

### CODE REVIEW

**Comment by sahmed95** [comment]
@souravsingh You shouldn't change the __init__ files as arokem warned and I guess it will be better if we work by opening an issue first and then fixing it so that there are no conflicts later with multiple people working on the same thing. All the absolute error issues are referenced in #969

**Comment by souravsingh** [comment]
@sahmed95 Sorry for this. I came to know this very late.

**Comment by sahmed95** [comment]
No issues. :smile: Even I had a doubt about this.

**Comment by sahmed95** [comment]
Also, I guess you can change segment too along with testing and tests here which will close #1000.

**Comment by souravsingh** [comment]
@sahmed95 I have done the segment fixing in the latest commit for the PR

**Comment by sahmed95** [comment]
Then I would suggest you update the name of the fix to "Use absolute imports in testing, tests and segment files

File: dipy/testing/__init__.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 IO_DATA_PATH = abspath(pjoin(dirname(__file__),
                              '..', 'io', 'tests', 'data'))
 
-from .spherepoints import sphere_points
-from .decorators import doctest_skip_parser
+from dipy.testing.spherepoints import sphere_points
```
Comment (arokem) [inline dipy/testing/__init__.py]: I'd stay away from `__init__` files, for now.

**Comment by arokem** [comment]
Hey @souravsingh - have you had a chance to take a look here? It would be great if you could revert the changes you made in the `__init__` file

**Comment by souravsingh** [comment]
My bad, I almost forgot about this.Thanks for reminding

**Comment by arokem** [comment]
This one needs a rebase.

**Comment by arokem** [comment]
Could you please rebase this on current master?

**Comment by souravsingh** [comment]
I am getting an error while rebasing `error: could not apply 6751bb5`

**Comment by souravsingh** [comment]
Alright
I got what was the conflict

**Comment by arokem** [comment]
Superseded by #1134. Closing

**Pull request closed without merging**